icarus_backend/clearance/ClearanceViews.py

```diff
@@ -0,0 +1,103 @@
+from .ClearanceModel import Clearance
+from django.utils import timezone
+from django.http import HttpResponse
+from django.core.serializers.json import DjangoJSONEncoder
+import json
+import uuid
+
+def add_clearance(request):
```

Comment:
Maybe change the name to update_clearance? The default state of missions should be a 'pending' clearance so you never really add it, you just change it. Also we will need to make sure this is all tied to the mission model before we approve this PR.